src/puppetlabs/ezbake/core.clj

```diff
@@ -418,6 +419,11 @@ Additional uberjar dependencies:
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Ephemeral Git Repo functions
 
+(defn get-git-sha
+  ;; Returns a git sha from the git repo in the current working directory
```

Comment:
Oh yeah, docstrings in Clojure are with `""`, so these should be like this: https://github.com/puppetlabs/ezbake/pull/421/files#diff-ff869252dfc543beb60d207e3a8a7727R338